### PR TITLE
zh: update security config requirement in source configuration file

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added, or deleted? (Required)

Add a description of the requirement of security config in `source-configuration file.md` to close https://github.com/pingcap/docs-dm/issues/607.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: zh/source-configuration-file.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ from:
 | `port` | 上游数据库的端口。|
 | `user` | 上游数据库使用的用户名。|
 | `password` | 上游数据库的用户密码。注意：推荐使用 dmctl 加密后的密码。|
-| `security` | 上游数据库 TLS 相关配置。|
+| `security` | 上游数据库 TLS 相关配置。配置的证书文件路径需为所有节点均可访问。若配置为本地路径，则集群所有节点需要将证书文件拷贝一份放在本机器相同的路径位置上。|
```
Comment (TomShawn) [inline zh/source-configuration-file.md:70]: Is this what you mean?
```suggestion
| `security` | 上游数据库 TLS 相关配置。配置的证书文件路径需能被所有节点访问。若配置为本地路径，则集群所有节点需要将证书文件拷贝一份放在节点机器相同的路径位置上。|
```

**Pull request merged**